styles/styles.css

```diff
@@ -0,0 +1,1423 @@
+@import url("./fonts.css");
+@import url('https://cdnjs.cloudflare.com/ajax/libs/flexboxgrid/6.3.1/flexboxgrid.min.css');
```

Comment:
Do not add unnecessary libraries into the project. Especially if you can't figure out how to use them.

Reply:
I applied the library to the block 4.